Logging.AspNetCore/RockLib.Logging.AspNetCore.Analyzers.Scenarios/RockLib.Logging.AspNetCore.Analyzers.Scenarios.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0</TargetFrameworks>
+		<TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+	</PropertyGroup>
+	<ItemGroup>
+	  <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
+	  <PackageReference Include="RockLib.Logging.AspNetCore" Version="3.2.4" />
```

Comment:
So are we going to remember this and come back and update it once we get around to updating RockLib.Logging?

Reply:
What are you referring to by "this"? I'm a bit confused....

Reply:
Sorry, the reference to `RockLib.Logging.AspNetCore`.  Since this reference was added I was wondering if we're going to need to come back to this project and update the version after we update RockLib.Logging.